src/hooks/useCart/useCart.tsx

```diff
@@ -235,11 +236,22 @@ export const useCart = (
    */
   const _completeCheckout = useCallback(() => {
     const complete = async (): Promise<void> => {
-      const transactions = Object.values(cart)
+      const allCleanedIdentifierInputs: IdentifierInput[] = [];
+      let transactions: any = [];
+      transactions = Object.values(cart)
         .filter(({ quantity }) => quantity)
         .map(({ category, quantity, identifierInputs }) => {
-          return { category, quantity, identifierInputs };
+          const cleanedIdentifierInputs = cleanIdentifierInputs(
+            identifierInputs
+          );
+          allCleanedIdentifierInputs.push(...cleanedIdentifierInputs);
+          return {
+            category,
+            quantity,
+            identifierInputs: cleanedIdentifierInputs,
+          };
         });
+
```

Comment:
nit: maybe can remove the additional empty lines!

Reply:
done!